dev/TreeView/TreeViewDragItemsCompletedEventArgs.cpp

```diff
@@ -19,4 +19,14 @@ DataPackageOperation TreeViewDragItemsCompletedEventArgs::DropResult() const
 winrt::IVectorView<winrt::IInspectable> TreeViewDragItemsCompletedEventArgs::Items()
 {
     return m_dragItemsCompletedEventArgs.Items();
-}
\ No newline at end of file
+}
+
+void TreeViewDragItemsCompletedEventArgs::NewParent(const winrt::IInspectable& parent)
+{
+    m_newParent = parent;
+}
```

Comment:
Would it be better to remove the setter and instead add a parameter to the constructor?